pints/toy/__init__.py

```diff
@@ -9,30 +9,30 @@
 from __future__ import absolute_import, division
 from __future__ import print_function, unicode_literals
 
-from ._toy_classes import ToyLogPDF, ToyModel, ToyODEModel          # noqa
```

Comment:
Just curious, why are these now removed the 'noqa'? (Or why was 'noqa' added before?)

Reply:
Because every single line in the file had a #noqa on it, I just removed this file from the style checking (like we already did with another similar page)